app/views/content/train-to-be-a-teacher/get-school-experience.md

```diff
@@ -17,8 +17,9 @@ calls_to_action:
           Use our service to search for and organise school experience in England.
         link_text: "Search for school experience"
         link_target: "https://schoolexperience.education.gov.uk/"
-promo_content:
-    - content/train-to-be-a-teacher/promos/mailing-list-promo
+cta_mailinglist:
+  mailinglist:
+    text: Find out about nearby events, what teaching is really like, and how to get one-to-one support.
```

Comment:
I know this is the wording in prod, but I wonder if it can be improved - starting with `nearby events` sounds a bit weird with no context? I'd be tempted to take out the reference to events unless you disagree and focus more on the bit about what teaching is like and how to get support (anything else?)